lib/cdo/cron_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'cdo/hip_chat'
+
+class CronHelper
+  CRON_REPORT_ROOM = 'cron-daily'
+  def self.report_cron_daily_result(task_name, success)
+    status = success ? 'success' : 'failure'
+    result_string = "#{task_name} : #{status}"
+    color = success ? 'green' : 'red'
```

Comment:
Any reason not to use the constants in `Slack::COLOR_MAP`?
https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/slack.rb#L5-L10